ui/css/logviewer.css

```diff
@@ -18,6 +17,13 @@ body {
     margin-bottom: 10px; /* Breathing space for run-data */
 }
 
+.navbar .dropdown-menu {
+    max-height: 600px;
+    overflow: auto;
+    margin-top: 10px;
+    padding-bottom: 8px;
+}
+
```

Comment:
Duplicated for Logviewer so it doesn't need the larger -navbar.css for just this one item.